pysen_ls/__main__.py

```diff
@@ -6,21 +6,24 @@
 _logger = logging.getLogger(__name__)
 
 
-def setup_logger() -> None:
-    handler = logging.FileHandler("pysen_ls.log", mode="w")
+def setup_logger(log_file: str) -> None:
```

Comment:
`log_file` accepts `None`, so I feel the correct type is `Optional[str]`

```suggestion
def setup_logger(log_file: Optional[str]) -> None:
```